docs/date.md

```diff
@@ -470,7 +470,7 @@ This is because months and years can be different lengths depending on which mon
 
 You can round the result using the `smallestUnit`, `roundingIncrement`, and `roundingMode` options.
 These behave as in the `Temporal.Duration.round()` method, but increments of days and larger are allowed.
-Since rounding to calendar units requires a reference point, `date` is used as the reference point.
+Since rounding to calendar units requires a reference point, `date` is used as the starting point.
```

Comment:
"Since" => "Because".  For clarity, IMHO we should only use "since" in the docs to refer to relative time.